torchsde/settings.py

```diff
@@ -23,16 +23,16 @@ def __str__(cls):
     def __contains__(cls, item):
         return item in cls.all()
 
+
 # TODO: consider moving all these enums into some appropriate section of the code, rather than having them be global
 #  like this. (e.g. instead set METHODS = {'euler': Euler, ...} in methods/__init__.py)
-
-
 class METHODS(metaclass=ContainerMeta):
     euler = 'euler'
     milstein = 'milstein'
     srk = 'srk'
     midpoint = 'midpoint'
     heun = 'heun'
+    log_ode_midpoint = 'log_ode_midpoint'
```

Comment:
The difference between midpoint and Heun here is essentially irrelevant. I'd like to add just `log_ode` as an alias for this. (And we can keep around `log_ode_midpoint` for the sake of some future `log_ode_heun`, if people really want that for some reason.)

Reply:
Done.